src/serializers/decoders/current/v9/decode_graph.c

```diff
@@ -36,6 +36,9 @@ static void _InitGraphDataStructure(Graph *g, uint64_t node_count, uint64_t edge
 	Graph_AllocateEdges(g, edge_count);
 	for(uint64_t i = 0; i < label_count; i++) Graph_AddLabel(g);
 	for(uint64_t i = 0; i < relation_count; i++) Graph_AddRelationType(g);
+	// flush adjacency and transposed adjacency matrices,
+	// guarantee matrix dimensions match graph's nodes count
+	Graph_GetAdjacencyMatrix(g, false);
```

Comment:
I suggest we keep on using `ApplyAll` to be future proff (in case we add additional matrices)